coding/cpp_style_guide.rst

```diff
@@ -84,10 +84,17 @@ These tasks are made much simpler if the code is easily readable and well-docume
 
 .. _style-guide-cpp-2-2:
 
-2-2. We are writing C++11/14 with some restrictions.
-----------------------------------------------------
+2-2. We are writing C++14
+-------------------------
 
-The official policy on the use of C++11 features is at :ref:`Policy on use of C++11/14 language features <style-guide-cpp-cpp-11-14>`.
+The C++11 standard and the C++14 improvements to it bring a number of useful language features that make the resulting code more expressive, easier to read, and safer.
+
+We follow the official: International Standard ISO/IEC 14882:2014(E) – Programming Language C++, without any compiler specific extensions.
```

Comment:
I think we should say that the minimum compiler level is gcc 6.3.1 (and clang x.x.x?) and that if there are divergences between what the standard says should be supported and what the compilers support, the intersection set of portable features is what we allow.

Reply:
Done